integration-tests/flipper/lib.rs

```diff
@@ -55,6 +55,10 @@ pub mod flipper {
     #[cfg(all(test, feature = "e2e-tests"))]
     mod e2e_tests {
         use super::*;
+        use ink_e2e::{
+            ChainBackend,
```

Comment:
flipper examples does not need `ChainBackend` import. Can you please double check other examples and keep only the necessary imports?

Reply:
good spot - I've removed `ChainBackend` import from overall 14 tests where it wasn't used